corehq/apps/userreports/tests/utils.py

```diff
@@ -108,3 +111,12 @@ def pre_run_with_es_backend(fn, *args, **kwargs):
         ),
     ]
 )
+
+
+def mock_sql_backend():
```

Comment:
Created these mocks to get past some tests that will require the SQL backend, but don't actually save the data source config to the db